package.json

```diff
@@ -89,6 +92,14 @@
     "webpack": "^5.74.0",
     "webpack-bundle-analyzer": "^4.6.1"
   },
+  "lint-staged": {
+    "*.{js,jsx,ts,tsx}": [
```

Comment:
This project does not use react/jsx, it uses vue

Reply:
fix that